core/dbt/main.py

```diff
@@ -211,7 +211,8 @@ def run_from_args(parsed):
     if task.config is not None:
         log_path = getattr(task.config, "log_path", None)
     log_manager.set_path(log_path)
-    setup_event_logger(log_path or "logs", "json", False, True)
+    # WHY WE SET DEBUG TO BE TRUE HERE previously?
```

Comment:
Changing one hard-code to another to make sure all tests passing. This file should be removed after feature branch merged